_shared_content/intelligence_center/integrations/anomali.md

```diff
@@ -0,0 +1,10 @@
+# External Integrations: Anomali ThreatStream
+
+SEKOIA's CTI feed is available in [Anomali's market place](https://www.anomali.com/marketplace/threat-intelligence-feeds), in the Threat Intelligence Feeds category
+
+Our integrations allows to see all the objects our feed produces along with the relationships between them.
+The PDF's of our reports can be downloaded in the attachment tab. 
+
+All our indicators are created as observables in the platform, and the sources are positionned as tags along with the kill chain phases.
+
+![Anomali's feed](/assets/intelligence_center/anomali_feed.png){: style="width: 100%; max-width: 100%"}
```

Comment:
```suggestion
![!Anomali's feed](/assets/intelligence_center/anomali_feed.png){: style="width: 100%; max-width: 100%"}
```

Small trick to allow users to open images in fullscreen.

Reply:
Nice one :-)